alf/algorithms/hypernetwork_algorithm.py

```diff
@@ -89,6 +89,13 @@ def __init__(self,
                  use_fc_bn=False,
                  num_particles=10,
                  entropy_regularization=1.,
+                 critic_optimizer=None,
+                 critic_hidden_layers=(100, 100),
+                 function_vi=False,
+                 function_bs=None,
```

Comment:
General comment about the particle-VI approaches that applies to the generator as well: I think these approaches should be detailed somewhere. I believe that only three people in total will understand what is happening in the amortized-particle-VI vs amortized-function-value-particle-VI vs ensemble-particle-VI. Possibly an incoming publication will describe them, but there should be a high-level description either here or in the alf docs.

Reply:
Good point, will do it after our icml submission.